agreement/pseudonode.go

```diff
@@ -484,7 +484,7 @@ func (t pseudonodeProposalsTask) execute(verifier *AsyncVoteVerifier, quit chan
 			Period:  uint64(vote.R.Period),
 		})
 	}
-	t.node.log.Infof("pseudonode.makeProposals: %v proposals created for round %v, period %v", len(verifiedVotes), t.round, t.period)
+	t.node.log.Infof("pseudonode.makeProposals: %v proposals created for round %d, period %d", len(verifiedVotes), t.round, t.period)
```

Comment:
The first `%v` here should be changed too, right?

Reply:
yes.